tasks/main.yaml

```diff
@@ -16,14 +9,40 @@
   when: rabbitmq_epel_install == true
   tags:
     - rabbitmq_pkg
+  when: ansible_os_family == "RedHat" and
+    ansible_distribution_major_version | int >= 6
+
+- name: Install rabbitmq-server (RedHat)
+  package:
+    name: "{{ rabbitmq_pkg }}"
+  notify:
+    - Restart rabbitmq-server
+  tags:
+    - rabbitmq_pkg
+  when: ansible_os_family == "RedHat" and
+    ansible_distribution_major_version | int >= 6
+
+- name: Add apt repository
+  apt_repository:
+    repo: deb http://www.rabbitmq.com/debian/ testing main
+    state: present
+  when: ansible_os_family == "Debian"
+
+- name: Add apt key
+  apt_key:
+    url: "{{ rabbitmq_debian_aptrepo_key }}"
+    state: "present"
+  when: ansible_os_family == "Debian"
 
-- name: Install rabbitmq-server
+- name: Install rabbitmq-server (Debian)
   package:
     name: "{{ rabbitmq_pkg }}"
+    update_cache: yes
```

Comment:
I kind of don't like to split package installation for RedHat and Debian. Please remove the above task called `Install rabbitmq-server (RedHat)` and remove the `update_cache` and `when` from this task.